### ISSUE TITLE
Uploaded images have no protection from being overwritten

### ISSUE THREAD

**Issue by DavidGriffith** [osCommerce/oscommerce2#374]
Suppose I have a product "cat frob" and I upload "frob.jpg" showing a nice cat frob.  Later one of my employees creates "dog frob" and uploads "frob.jpg".  As is, this will cause oscommerce to happily replace the picture of a dog frob to replace the picture of a cat frob with no warning.  This is a Bad Thing.  There needs to be some mechanism to check if that filename exists and complain if there's a collision or do something like insert an identifying string into the filename like this: "frob-12345.jpg" where "12345" is an internal product number.

**Comment by MOPS-DanCole** [comment]
I disagree...when I want to update an image I just want to replace it and be done with it...I use an easy to understand naming convention related to the product so I don't see the need for this or necessity to build it into the core but keep in mind that you can always customize the code anyway you want.
